src/cmds/play.c

```diff
@@ -151,7 +187,11 @@ int main(int argc, char **argv) {
 
 		printf("Progress:\n");
 
-		_bl = min(fread(_fbuffer, 1, 64 * 1024 * 1024, fd), _bl);
+#if PLAY_FROM_FLASH
```

Comment:
We can get rid of this `#ifdef` by moving `_bl = ...` to the previous `#ifdef` as it's not used between this macros.